javascript/core/recipes-i/js-practice-math-round.md

```diff
@@ -16,6 +16,10 @@ standards:
   javascript.standard-library.1: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```

Comment:
This is `introduction`, most languages have this function, so I don't think it makes sense as a `workout`.